### DISCUSSION TITLE
How to undo commit of a file edited on Github?

### DISCUSSION THREAD

**Discussion by rishumehra** [community/community#21749] [New to GitHub]
<p>Hi Folks,</p>
<p>I made some changes on a MD file with in GitHub editor and commited the file too.</p>
<p>But now I want to undo those changes, and rever back the file(s) as it was before. Is there any possibility like that?</p>
<p>Please help regarding the same.</p>
<p>Cheers,</p>
<p>Rishu Mehra</p>

**Comment by mpboom** [comment, marked as answer]
<p>Did you commit the file trough a pull request? In that case there should be a  <strong>Revert merge</strong> button on the (closed) pull request.</p>
